src/app/debug/filter-side-drawer/filter-side-drawer.component.ts

```diff
@@ -24,14 +24,14 @@ import { TitleCasePipe } from '@angular/common';
 })
 export class FilterSideDrawerComponent implements OnDestroy, OnInit {
   protected shouldShowFilter!: boolean;
-  protected metadataNames!: string[];
-  protected filters: Map<string, string> = new Map<string, string>();
+  protected metadataLabels!: string[];
   protected currentRecords: Map<string, Array<string>> = new Map<string, Array<string>>();
+  protected metadataTypes!: Map<string, string>;
 
   shouldShowFilterSubscriber!: Subscription;
-  metadataNamesSubscriber!: Subscription;
-  filterSubscriber!: Subscription;
+  metadataLabelsSubscriber!: Subscription;
   currentRecordsSubscriber!: Subscription;
+  metadataTypesSubscriber!: Subscription;
```

Comment:
If I don't remove the undefined from the subscription, a check for each subscription would have to be made for each subscription seperately to see if it's undefined, otherwise it will give an error.
Did you have something in mind for that? I think the same reason goes for why the other subscription remove undefined as well.

Reply:
Well you'd get: 
```typescript
unsubscribeAll(): void {
  this.shouldShowFilterSubscribtion?.unsubscribe();
  this.metadataLabelsSubscribtion?.unsubscribe();
  this.currentRecordsSubscribtion?.unsubscribe();
  this.metadataTypesSubscribtion?.unsubscribe();
}
```

But I think it is better than having an object that can be null after the constructor, but saying it isn't. It is also generally good to try to avoid it as much as possible, just so no wrong assumptions will go through.

Goed geëngelst.

Reply:
Alright. I added the change.